clarity_ext/domain/container.py

```diff
@@ -91,18 +91,25 @@ def __init__(self, mapping=None, container_type=None):
         else:
             raise ValueError("Unknown plate type '{}'".format(self.container_type))
 
+        if self.resource:
+            self.id = self.resource.id
+
+    def __repr__(self):
+        return "<Container id={}>".format(self.id)
+
     @classmethod
     def create_from_rest_resource(cls, resource, artifacts):
         """
         Creates a container based on a resource from the REST API.
         """
-        # TODO: This needs to be cleaned up, i.e. how exactly the rest resource and domain objects should interact
         if resource.type.name == "96 well plate":
             ret = Container(container_type=Container.CONTAINER_TYPE_96_WELLS_PLATE)
+            ret.size = PlateSize(width=resource.type.x_dimension["size"], height=resource.type.y_dimension["size"])
```

Comment:
Side note: It looks like sthlm genologics has the notion of x_dimension and y_dimension. If I've misunderstood this, so that it's our code using this notion, we should change it.